### PR TITLE
[EDITORIAL] Column doc link formatting fix

### PR DESCRIPTION
**Pull request by AWS-ZachErdman**
We inconsistently wrote the names of columns in the links that link to columns throughout the spec. Sometimes they would have a space between words and sometimes not. I adjusted all column links to have no spacing between words to make it clear it was a column.

I also fixed a link to ContractedUnitPrice that incorrectly linked to the glossary.

### CODE REVIEW

File: specification/columns/billedcost.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
+The [*billed cost*](#glossary:billed-cost) represents a charge serving as the basis for invoicing, inclusive of the impacts of all reduced rates and discounts while excluding the [*amortization*](#glossary:amortization) of relevant purchases (one-time or recurring) paid to cover future eligible charges. This cost is denominated in the [BillingCurrency](#billingcurrency). The Billed Cost is commonly used to perform FinOps capabilities that require cash-basis accounting such as cost allocation, budgeting, and invoice reconciliation.
 
 The BilledCost column MUST be present in the billing data and MUST NOT be null. This column MUST be of type Decimal, MUST conform to [Numeric Format](#numericformat), and be denominated in the BillingCurrency. The sum of the BilledCost for [*rows*](#glossary:row) in a given [*billing period*](#glossary:billing-period) MUST match the sum of the invoices received for that *billing period* for a [*billing account*](#glossary:billing-account).
```
Comment (jpradocueva) [inline specification/columns/billedcost.md:5]: What about `Numeric Format` listed in this line?

**Review by jpradocueva — COMMENTED** [review]
Should `Numeric Format` be updated?

**Comment by AWS-ZachErdman** [comment]
> Should `Numeric Format` be updated?

I don't think that's needed since it's not a column in FOCUS, so it wouldn't appear as NumericFormat.

**Comment by jpradocueva** [comment]
@AWS-ZachErdman The group discussed this topic and indicated that it was intentionally defined with a blank space when the column is not listed as a Mandatory requirement.

**Comment by AWS-ZachErdman** [comment]
I see - when the column is being referenced in a requirement, it's supposed to not have a space.

**Comment by jpradocueva** [comment]
Extracted from the Members' Group, Meeting Minutes, May 23.
>Some members noted that certain names were intentionally defined with a space, such as "Billing Currency" instead of "BillingCurrency." This distinction helps differentiate when the column name is used in a general context versus the column Name ID, which is employed in normative statements.

**Comment by jpradocueva** [comment]
### Action Items from TF-1 call on October 1st: 
- [ ] [TF1-#469](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/469) Joaquin, @jpradocueva : Prepare editorial guidelines reflecting agreed column naming conventions.

**Comment by jpradocueva** [comment]
PR #582 contains editorial guidelines to resolve this issue.

**Pull request closed without merging**